Inliner crash

## Compiler version

v3.0.1-RC1
## Minimized code
Two separate files:

**DFType.scala**
```Scala
import scala.quoted.*

class InlinedInt[T <: Int]

object DFType:
  trait Width[T]:
    type Out <: Int
  transparent inline given [T]: Width[T] = ${ getWidthMacro[T] }
  def getWidthMacro[T](using Quotes, Type[T]): Expr[Width[T]] =
    '{
      new Width[T] {
        type Out = 1
      }
    }

  trait DFBits[W <: Int]
  object DFBits extends DFBitsCompanion

trait DFToken[T]
extension [T](token: DFToken[T])(using w: DFType.Width[T])
  def width: InlinedInt[w.Out] = ???
```

**DFBits.scala**
```scala
trait DFBitsCompanion:
  type Token[W <: Int] = DFToken[DFType.DFBits[W]]
  extension [LW <: Int](lhs: DFType.DFBits.Token[LW]) def foo: Unit = lhs.width
```

## Output (click arrow to expand)
<details>

```scala
[error] java.lang.NoClassDefFoundError: DFBitsCompanion
[error] java.base/java.lang.ClassLoader.defineClass1(Native Method)
[error] java.base/java.lang.ClassLoader.defineClass(ClassLoader.java:1016)
[error] java.base/java.security.SecureClassLoader.defineClass(SecureClassLoader.java:174)
[error] java.base/java.net.URLClassLoader.defineClass(URLClassLoader.java:550)
[error] java.base/java.net.URLClassLoader$1.run(URLClassLoader.java:458)
[error] java.base/java.net.URLClassLoader$1.run(URLClassLoader.java:452)
[error] java.base/java.security.AccessController.doPrivileged(Native Method)
[error] java.base/java.net.URLClassLoader.findClass(URLClassLoader.java:451)
[error] java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:588)
[error] java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:521)
[error] java.base/java.lang.Class.getDeclaredFields0(Native Method)
[error] java.base/java.lang.Class.privateGetDeclaredFields(Class.java:3061)
[error] java.base/java.lang.Class.getField0(Class.java:3246)
[error] java.base/java.lang.Class.getField(Class.java:1997)
[error] dotty.tools.dotc.transform.Splicer$Interpreter.loadModule(Splicer.scala:387)
[error] dotty.tools.dotc.transform.Splicer$Interpreter.interpretedStaticMethodCall(Splicer.scala:356)
[error] dotty.tools.dotc.transform.Splicer$Interpreter.interpretTree(Splicer.scala:260)
[error] dotty.tools.dotc.transform.Splicer$Interpreter.interpretTree$$anonfun$2(Splicer.scala:281)
[error] dotty.tools.dotc.transform.Splicer$.$anonfun$2(Splicer.scala:56)
[error] scala.Option.fold(Option.scala:263)
[error] dotty.tools.dotc.transform.Splicer$.splice(Splicer.scala:56)
[error] dotty.tools.dotc.typer.Inliner.dotty$tools$dotc$typer$Inliner$$expandMacro(Inliner.scala:1633)
[error] dotty.tools.dotc.typer.Inliner$InlineTyper.typedApply(Inliner.scala:1414)
[error] dotty.tools.dotc.typer.Typer.typedUnnamed$1(Typer.scala:2695)
[error] dotty.tools.dotc.typer.Typer.typedUnadapted(Typer.scala:2758)
[error] dotty.tools.dotc.typer.ReTyper.typedUnadapted(ReTyper.scala:118)
[error] dotty.tools.dotc.typer.Inliner$InlineTyper.typedUnadapted(Inliner.scala:1494)
[error] dotty.tools.dotc.typer.Typer.typed(Typer.scala:2823)
[error] dotty.tools.dotc.typer.Typer.typed(Typer.scala:2820)
[error] dotty.tools.dotc.typer.Typer.typed(Typer.scala:2827)
[error] dotty.tools.dotc.typer.Inliner.inlined(Inliner.scala:901)
[error] dotty.tools.dotc.typer.Inliner$.inlineCall(Inliner.scala:153)
[error] dotty.tools.dotc.typer.Typer.adaptNoArgsOther$4(Typer.scala:3504)
[error] dotty.tools.dotc.typer.Typer.adaptNoArgs$1(Typer.scala:3620)
[error] dotty.tools.dotc.typer.Typer.adapt1(Typer.scala:3811)
[error] dotty.tools.dotc.typer.Typer.adapt(Typer.scala:3163)
[error] dotty.tools.dotc.typer.Typer.readapt$1(Typer.scala:3174)
[error] dotty.tools.dotc.typer.Typer.adapt1(Typer.scala:3798)
[error] dotty.tools.dotc.typer.Typer.adapt(Typer.scala:3163)
[error] dotty.tools.dotc.typer.Implicits.typedImplicit(Implicits.scala:1024)
[error] dotty.tools.dotc.typer.Implicits.typedImplicit$(Implicits.scala:769)
[error] dotty.tools.dotc.typer.Typer.typedImplicit(Typer.scala:106)
[error] dotty.tools.dotc.typer.Implicits$ImplicitSearch.tryImplicit(Implicits.scala:1119)
[error] dotty.tools.dotc.typer.Implicits$ImplicitSearch.rank$1(Implicits.scala:1191)
[error] dotty.tools.dotc.typer.Implicits$ImplicitSearch.searchImplicit(Implicits.scala:1359)
[error] dotty.tools.dotc.typer.Implicits$ImplicitSearch.searchImplicit(Implicits.scala:1366)
[error] dotty.tools.dotc.typer.Implicits$ImplicitSearch.searchImplicit(Implicits.scala:1374)
[error] dotty.tools.dotc.typer.Implicits$ImplicitSearch.bestImplicit(Implicits.scala:1399)
[error] dotty.tools.dotc.typer.Implicits.inferImplicit(Implicits.scala:974)
[error] dotty.tools.dotc.typer.Implicits.inferImplicit$(Implicits.scala:769)
[error] dotty.tools.dotc.typer.Typer.inferImplicit(Typer.scala:106)
[error] dotty.tools.dotc.typer.Implicits.inferImplicitArg(Implicits.scala:843)
[error] dotty.tools.dotc.typer.Implicits.inferImplicitArg$(Implicits.scala:769)
[error] dotty.tools.dotc.typer.Typer.inferImplicitArg(Typer.scala:106)
[error] dotty.tools.dotc.typer.Typer.implicitArgs$1(Typer.scala:3285)
[error] dotty.tools.dotc.typer.Typer.addImplicitArgs$3(Typer.scala:3321)
[error] dotty.tools.dotc.typer.Typer.adaptNoArgsImplicitMethod$2(Typer.scala:3401)
[error] dotty.tools.dotc.typer.Typer.adaptNoArgs$1(Typer.scala:3598)
[error] dotty.tools.dotc.typer.Typer.adapt1(Typer.scala:3811)
[error] dotty.tools.dotc.typer.Typer.adapt(Typer.scala:3163)
[error] dotty.tools.dotc.typer.Typer.typed(Typer.scala:2823)
[error] dotty.tools.dotc.typer.Applications.extMethodApply(Applications.scala:2225)
[error] dotty.tools.dotc.typer.Applications.extMethodApply$(Applications.scala:316)
[error] dotty.tools.dotc.typer.Typer.extMethodApply(Typer.scala:106)
[error] dotty.tools.dotc.typer.Typer.tryExtension$1(Typer.scala:3077)
[error] dotty.tools.dotc.typer.Typer.tryExtensionOrConversion(Typer.scala:3088)
[error] dotty.tools.dotc.typer.Typer.typedSelect(Typer.scala:583)
[error] dotty.tools.dotc.typer.Typer.typeSelectOnTerm$1(Typer.scala:622)
[error] dotty.tools.dotc.typer.Typer.typedSelect(Typer.scala:659)
[error] dotty.tools.dotc.typer.Typer.typedNamed$1(Typer.scala:2665)
[error] dotty.tools.dotc.typer.Typer.typedUnadapted(Typer.scala:2757)
[error] dotty.tools.dotc.typer.Typer.typed(Typer.scala:2823)
[error] dotty.tools.dotc.typer.Typer.typed(Typer.scala:2827)
[error] dotty.tools.dotc.typer.Typer.typedExpr(Typer.scala:2943)
[error] dotty.tools.dotc.typer.Typer.$anonfun$34(Typer.scala:2168)
[error] dotty.tools.dotc.typer.PrepareInlineable$.dropInlineIfError(PrepareInlineable.scala:225)
[error] dotty.tools.dotc.typer.Typer.typedDefDef(Typer.scala:2168)
[error] dotty.tools.dotc.typer.Typer.typedNamed$1(Typer.scala:2672)
[error] dotty.tools.dotc.typer.Typer.typedUnadapted(Typer.scala:2757)
[error] dotty.tools.dotc.typer.Typer.typed(Typer.scala:2823)
[error] dotty.tools.dotc.typer.Typer.typed(Typer.scala:2827)
[error] dotty.tools.dotc.typer.Typer.traverse$1(Typer.scala:2849)
[error] dotty.tools.dotc.typer.Typer.typedStats(Typer.scala:2899)
[error] dotty.tools.dotc.typer.Typer.typedClassDef(Typer.scala:2354)
[error] dotty.tools.dotc.typer.Typer.typedTypeOrClassDef$2(Typer.scala:2683)
[error] dotty.tools.dotc.typer.Typer.typedNamed$1(Typer.scala:2687)
[error] dotty.tools.dotc.typer.Typer.typedUnadapted(Typer.scala:2757)
[error] dotty.tools.dotc.typer.Typer.typed(Typer.scala:2823)
[error] dotty.tools.dotc.typer.Typer.typed(Typer.scala:2827)
[error] dotty.tools.dotc.typer.Typer.traverse$1(Typer.scala:2849)
[error] dotty.tools.dotc.typer.Typer.typedStats(Typer.scala:2899)
[error] dotty.tools.dotc.typer.Typer.typedPackageDef(Typer.scala:2477)
[error] dotty.tools.dotc.typer.Typer.typedUnnamed$1(Typer.scala:2728)
[error] dotty.tools.dotc.typer.Typer.typedUnadapted(Typer.scala:2758)
[error] dotty.tools.dotc.typer.Typer.typed(Typer.scala:2823)
[error] dotty.tools.dotc.typer.Typer.typed(Typer.scala:2827)
[error] dotty.tools.dotc.typer.Typer.traverse$1(Typer.scala:2876)
[error] dotty.tools.dotc.typer.Typer.typedStats(Typer.scala:2899)
[error] dotty.tools.dotc.typer.Typer.typedPackageDef(Typer.scala:2477)
[error] dotty.tools.dotc.typer.Typer.typedUnnamed$1(Typer.scala:2728)
[error] dotty.tools.dotc.typer.Typer.typedUnadapted(Typer.scala:2758)
[error] dotty.tools.dotc.typer.Typer.typed(Typer.scala:2823)
[error] dotty.tools.dotc.typer.Typer.typed(Typer.scala:2827)
[error] dotty.tools.dotc.typer.Typer.typedExpr(Typer.scala:2943)
[error] dotty.tools.dotc.typer.FrontEnd.liftedTree1$1(FrontEnd.scala:79)
[error] dotty.tools.dotc.typer.FrontEnd.typeCheck$$anonfun$1(FrontEnd.scala:84)
[error] dotty.tools.dotc.typer.FrontEnd.monitor(FrontEnd.scala:43)
[error] dotty.tools.dotc.typer.FrontEnd.typeCheck(FrontEnd.scala:85)
[error] dotty.tools.dotc.typer.FrontEnd.runOn$$anonfun$3(FrontEnd.scala:120)
[error] scala.runtime.function.JProcedure1.apply(JProcedure1.java:15)
[error] scala.runtime.function.JProcedure1.apply(JProcedure1.java:10)
[error] scala.collection.immutable.List.foreach(List.scala:333)
[error] dotty.tools.dotc.typer.FrontEnd.runOn(FrontEnd.scala:120)
[error] dotty.tools.dotc.Run.runPhases$4$$anonfun$4(Run.scala:205)
[error] scala.runtime.function.JProcedure1.apply(JProcedure1.java:15)
[error] scala.runtime.function.JProcedure1.apply(JProcedure1.java:10)
[error] scala.collection.ArrayOps$.foreach$extension(ArrayOps.scala:1323)
[error] dotty.tools.dotc.Run.runPhases$5(Run.scala:216)
[error] dotty.tools.dotc.Run.compileUnits$$anonfun$1(Run.scala:224)
[error] scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.scala:18)
[error] dotty.tools.dotc.util.Stats$.maybeMonitored(Stats.scala:67)
[error] dotty.tools.dotc.Run.compileUnits(Run.scala:231)
[error] dotty.tools.dotc.Run.compileUnits(Run.scala:172)
[error] dotty.tools.dotc.Driver.finish(Driver.scala:60)
[error] dotty.tools.dotc.Driver.doCompile(Driver.scala:40)
[error] dotty.tools.xsbt.CompilerBridgeDriver.run(CompilerBridgeDriver.java:88)
[error] dotty.tools.xsbt.CompilerBridge.run(CompilerBridge.java:22)
[error] sbt.internal.inc.AnalyzingCompiler.compile(AnalyzingCompiler.scala:91)
[error] sbt.internal.inc.MixedAnalyzingCompiler.$anonfun$compile$7(MixedAnalyzingCompiler.scala:186)
[error] scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
[error] sbt.internal.inc.MixedAnalyzingCompiler.timed(MixedAnalyzingCompiler.scala:241)
[error] sbt.internal.inc.MixedAnalyzingCompiler.$anonfun$compile$4(MixedAnalyzingCompiler.scala:176)
[error] sbt.internal.inc.MixedAnalyzingCompiler.$anonfun$compile$4$adapted(MixedAnalyzingCompiler.scala:157)
[error] sbt.internal.inc.JarUtils$.withPreviousJar(JarUtils.scala:239)
[error] sbt.internal.inc.MixedAnalyzingCompiler.compileScala$1(MixedAnalyzingCompiler.scala:157)
[error] sbt.internal.inc.MixedAnalyzingCompiler.compile(MixedAnalyzingCompiler.scala:204)
[error] sbt.internal.inc.IncrementalCompilerImpl.$anonfun$compileInternal$1(IncrementalCompilerImpl.scala:528)
[error] sbt.internal.inc.IncrementalCompilerImpl.$anonfun$compileInternal$1$adapted(IncrementalCompilerImpl.scala:528)
[error] sbt.internal.inc.Incremental$.$anonfun$apply$5(Incremental.scala:174)
[error] sbt.internal.inc.Incremental$.$anonfun$apply$5$adapted(Incremental.scala:172)
[error] sbt.internal.inc.Incremental$$anon$2.run(Incremental.scala:457)
[error] sbt.internal.inc.IncrementalCommon$CycleState.next(IncrementalCommon.scala:116)
[error] sbt.internal.inc.IncrementalCommon$$anon$1.next(IncrementalCommon.scala:56)
[error] sbt.internal.inc.IncrementalCommon$$anon$1.next(IncrementalCommon.scala:52)
[error] sbt.internal.inc.IncrementalCommon.cycle(IncrementalCommon.scala:261)
[error] sbt.internal.inc.Incremental$.$anonfun$incrementalCompile$8(Incremental.scala:412)
[error] sbt.internal.inc.Incremental$.withClassfileManager(Incremental.scala:499)
[error] sbt.internal.inc.Incremental$.incrementalCompile(Incremental.scala:399)
[error] sbt.internal.inc.Incremental$.apply(Incremental.scala:166)
[error] sbt.internal.inc.IncrementalCompilerImpl.compileInternal(IncrementalCompilerImpl.scala:528)
[error] sbt.internal.inc.IncrementalCompilerImpl.$anonfun$compileIncrementally$1(IncrementalCompilerImpl.scala:482)
[error] sbt.internal.inc.IncrementalCompilerImpl.handleCompilationError(IncrementalCompilerImpl.scala:332)
[error] sbt.internal.inc.IncrementalCompilerImpl.compileIncrementally(IncrementalCompilerImpl.scala:420)
[error] sbt.internal.inc.IncrementalCompilerImpl.compile(IncrementalCompilerImpl.scala:137)
[error] sbt.Defaults$.compileIncrementalTaskImpl(Defaults.scala:2346)
[error] sbt.Defaults$.$anonfun$compileIncrementalTask$2(Defaults.scala:2303)
[error] sbt.internal.io.Retry$.apply(Retry.scala:46)
[error] sbt.internal.io.Retry$.apply(Retry.scala:28)
[error] sbt.internal.io.Retry$.apply(Retry.scala:23)
[error] sbt.internal.server.BspCompileTask$.compute(BspCompileTask.scala:31)
[error] sbt.Defaults$.$anonfun$compileIncrementalTask$1(Defaults.scala:2299)
[error] scala.Function1.$anonfun$compose$1(Function1.scala:49)
[error] sbt.internal.util.$tilde$greater.$anonfun$$u2219$1(TypeFunctions.scala:62)
[error] sbt.std.Transform$$anon$4.work(Transform.scala:68)
[error] sbt.Execute.$anonfun$submit$2(Execute.scala:282)
[error] sbt.internal.util.ErrorHandling$.wideConvert(ErrorHandling.scala:23)
[error] sbt.Execute.work(Execute.scala:291)
[error] sbt.Execute.$anonfun$submit$1(Execute.scala:282)
[error] sbt.ConcurrentRestrictions$$anon$4.$anonfun$submitValid$1(ConcurrentRestrictions.scala:265)
[error] sbt.CompletionService$$anon$2.call(CompletionService.scala:64)
[error] java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[error] java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
[error] java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[error] java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[error] java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[error] java.base/java.lang.Thread.run(Thread.java:834)
[error]            
[error] java.lang.NoClassDefFoundError: DFBitsCompanion
[error] 	at java.base/java.lang.ClassLoader.defineClass1(Native Method)
[error] 	at java.base/java.lang.ClassLoader.defineClass(ClassLoader.java:1016)
[error] 	at java.base/java.security.SecureClassLoader.defineClass(SecureClassLoader.java:174)
[error] 	at java.base/java.net.URLClassLoader.defineClass(URLClassLoader.java:550)
[error] 	at java.base/java.net.URLClassLoader$1.run(URLClassLoader.java:458)
[error] 	at java.base/java.net.URLClassLoader$1.run(URLClassLoader.java:452)
[error] 	at java.base/java.security.AccessController.doPrivileged(Native Method)
[error] 	at java.base/java.net.URLClassLoader.findClass(URLClassLoader.java:451)
[error] 	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:588)
[error] 	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:521)
[error] 	at java.base/java.lang.Class.getDeclaredFields0(Native Method)
[error] 	at java.base/java.lang.Class.privateGetDeclaredFields(Class.java:3061)
[error] 	at java.base/java.lang.Class.getField0(Class.java:3246)
[error] 	at java.base/java.lang.Class.getField(Class.java:1997)
[error] 	at dotty.tools.dotc.transform.Splicer$Interpreter.loadModule(Splicer.scala:387)
[error] 	at dotty.tools.dotc.transform.Splicer$Interpreter.interpretedStaticMethodCall(Splicer.scala:356)
[error] 	at dotty.tools.dotc.transform.Splicer$Interpreter.interpretTree(Splicer.scala:260)
[error] 	at dotty.tools.dotc.transform.Splicer$Interpreter.interpretTree$$anonfun$2(Splicer.scala:281)
[error] 	at dotty.tools.dotc.transform.Splicer$.$anonfun$2(Splicer.scala:56)
[error] 	at scala.Option.fold(Option.scala:263)
[error] 	at dotty.tools.dotc.transform.Splicer$.splice(Splicer.scala:56)
[error] 	at dotty.tools.dotc.typer.Inliner.dotty$tools$dotc$typer$Inliner$$expandMacro(Inliner.scala:1633)
[error] 	at dotty.tools.dotc.typer.Inliner$InlineTyper.typedApply(Inliner.scala:1414)
[error] 	at dotty.tools.dotc.typer.Typer.typedUnnamed$1(Typer.scala:2695)
[error] 	at dotty.tools.dotc.typer.Typer.typedUnadapted(Typer.scala:2758)
[error] 	at dotty.tools.dotc.typer.ReTyper.typedUnadapted(ReTyper.scala:118)
[error] 	at dotty.tools.dotc.typer.Inliner$InlineTyper.typedUnadapted(Inliner.scala:1494)
[error] 	at dotty.tools.dotc.typer.Typer.typed(Typer.scala:2823)
[error] 	at dotty.tools.dotc.typer.Typer.typed(Typer.scala:2820)
[error] 	at dotty.tools.dotc.typer.Typer.typed(Typer.scala:2827)
[error] 	at dotty.tools.dotc.typer.Inliner.inlined(Inliner.scala:901)
[error] 	at dotty.tools.dotc.typer.Inliner$.inlineCall(Inliner.scala:153)
[error] 	at dotty.tools.dotc.typer.Typer.adaptNoArgsOther$4(Typer.scala:3504)
[error] 	at dotty.tools.dotc.typer.Typer.adaptNoArgs$1(Typer.scala:3620)
[error] 	at dotty.tools.dotc.typer.Typer.adapt1(Typer.scala:3811)
[error] 	at dotty.tools.dotc.typer.Typer.adapt(Typer.scala:3163)
[error] 	at dotty.tools.dotc.typer.Typer.readapt$1(Typer.scala:3174)
[error] 	at dotty.tools.dotc.typer.Typer.adapt1(Typer.scala:3798)
[error] 	at dotty.tools.dotc.typer.Typer.adapt(Typer.scala:3163)
[error] 	at dotty.tools.dotc.typer.Implicits.typedImplicit(Implicits.scala:1024)
[error] 	at dotty.tools.dotc.typer.Implicits.typedImplicit$(Implicits.scala:769)
[error] 	at dotty.tools.dotc.typer.Typer.typedImplicit(Typer.scala:106)
[error] 	at dotty.tools.dotc.typer.Implicits$ImplicitSearch.tryImplicit(Implicits.scala:1119)
[error] 	at dotty.tools.dotc.typer.Implicits$ImplicitSearch.rank$1(Implicits.scala:1191)
[error] 	at dotty.tools.dotc.typer.Implicits$ImplicitSearch.searchImplicit(Implicits.scala:1359)
[error] 	at dotty.tools.dotc.typer.Implicits$ImplicitSearch.searchImplicit(Implicits.scala:1366)
[error] 	at dotty.tools.dotc.typer.Implicits$ImplicitSearch.searchImplicit(Implicits.scala:1374)
[error] 	at dotty.tools.dotc.typer.Implicits$ImplicitSearch.bestImplicit(Implicits.scala:1399)
[error] 	at dotty.tools.dotc.typer.Implicits.inferImplicit(Implicits.scala:974)
[error] 	at dotty.tools.dotc.typer.Implicits.inferImplicit$(Implicits.scala:769)
[error] 	at dotty.tools.dotc.typer.Typer.inferImplicit(Typer.scala:106)
[error] 	at dotty.tools.dotc.typer.Implicits.inferImplicitArg(Implicits.scala:843)
[error] 	at dotty.tools.dotc.typer.Implicits.inferImplicitArg$(Implicits.scala:769)
[error] 	at dotty.tools.dotc.typer.Typer.inferImplicitArg(Typer.scala:106)
[error] 	at dotty.tools.dotc.typer.Typer.implicitArgs$1(Typer.scala:3285)
[error] 	at dotty.tools.dotc.typer.Typer.addImplicitArgs$3(Typer.scala:3321)
[error] 	at dotty.tools.dotc.typer.Typer.adaptNoArgsImplicitMethod$2(Typer.scala:3401)
[error] 	at dotty.tools.dotc.typer.Typer.adaptNoArgs$1(Typer.scala:3598)
[error] 	at dotty.tools.dotc.typer.Typer.adapt1(Typer.scala:3811)
[error] 	at dotty.tools.dotc.typer.Typer.adapt(Typer.scala:3163)
[error] 	at dotty.tools.dotc.typer.Typer.typed(Typer.scala:2823)
[error] 	at dotty.tools.dotc.typer.Applications.extMethodApply(Applications.scala:2225)
[error] 	at dotty.tools.dotc.typer.Applications.extMethodApply$(Applications.scala:316)
[error] 	at dotty.tools.dotc.typer.Typer.extMethodApply(Typer.scala:106)
[error] 	at dotty.tools.dotc.typer.Typer.tryExtension$1(Typer.scala:3077)
[error] 	at dotty.tools.dotc.typer.Typer.tryExtensionOrConversion(Typer.scala:3088)
[error] 	at dotty.tools.dotc.typer.Typer.typedSelect(Typer.scala:583)
[error] 	at dotty.tools.dotc.typer.Typer.typeSelectOnTerm$1(Typer.scala:622)
[error] 	at dotty.tools.dotc.typer.Typer.typedSelect(Typer.scala:659)
[error] 	at dotty.tools.dotc.typer.Typer.typedNamed$1(Typer.scala:2665)
[error] 	at dotty.tools.dotc.typer.Typer.typedUnadapted(Typer.scala:2757)
[error] 	at dotty.tools.dotc.typer.Typer.typed(Typer.scala:2823)
[error] 	at dotty.tools.dotc.typer.Typer.typed(Typer.scala:2827)
[error] 	at dotty.tools.dotc.typer.Typer.typedExpr(Typer.scala:2943)
[error] 	at dotty.tools.dotc.typer.Typer.$anonfun$34(Typer.scala:2168)
[error] 	at dotty.tools.dotc.typer.PrepareInlineable$.dropInlineIfError(PrepareInlineable.scala:225)
[error] 	at dotty.tools.dotc.typer.Typer.typedDefDef(Typer.scala:2168)
[error] 	at dotty.tools.dotc.typer.Typer.typedNamed$1(Typer.scala:2672)
[error] 	at dotty.tools.dotc.typer.Typer.typedUnadapted(Typer.scala:2757)
[error] 	at dotty.tools.dotc.typer.Typer.typed(Typer.scala:2823)
[error] 	at dotty.tools.dotc.typer.Typer.typed(Typer.scala:2827)
[error] 	at dotty.tools.dotc.typer.Typer.traverse$1(Typer.scala:2849)
[error] 	at dotty.tools.dotc.typer.Typer.typedStats(Typer.scala:2899)
[error] 	at dotty.tools.dotc.typer.Typer.typedClassDef(Typer.scala:2354)
[error] 	at dotty.tools.dotc.typer.Typer.typedTypeOrClassDef$2(Typer.scala:2683)
[error] 	at dotty.tools.dotc.typer.Typer.typedNamed$1(Typer.scala:2687)
[error] 	at dotty.tools.dotc.typer.Typer.typedUnadapted(Typer.scala:2757)
[error] 	at dotty.tools.dotc.typer.Typer.typed(Typer.scala:2823)
[error] 	at dotty.tools.dotc.typer.Typer.typed(Typer.scala:2827)
[error] 	at dotty.tools.dotc.typer.Typer.traverse$1(Typer.scala:2849)
[error] 	at dotty.tools.dotc.typer.Typer.typedStats(Typer.scala:2899)
[error] 	at dotty.tools.dotc.typer.Typer.typedPackageDef(Typer.scala:2477)
[error] 	at dotty.tools.dotc.typer.Typer.typedUnnamed$1(Typer.scala:2728)
[error] 	at dotty.tools.dotc.typer.Typer.typedUnadapted(Typer.scala:2758)
[error] 	at dotty.tools.dotc.typer.Typer.typed(Typer.scala:2823)
[error] 	at dotty.tools.dotc.typer.Typer.typed(Typer.scala:2827)
[error] 	at dotty.tools.dotc.typer.Typer.traverse$1(Typer.scala:2876)
[error] 	at dotty.tools.dotc.typer.Typer.typedStats(Typer.scala:2899)
[error] 	at dotty.tools.dotc.typer.Typer.typedPackageDef(Typer.scala:2477)
[error] 	at dotty.tools.dotc.typer.Typer.typedUnnamed$1(Typer.scala:2728)
[error] 	at dotty.tools.dotc.typer.Typer.typedUnadapted(Typer.scala:2758)
[error] 	at dotty.tools.dotc.typer.Typer.typed(Typer.scala:2823)
[error] 	at dotty.tools.dotc.typer.Typer.typed(Typer.scala:2827)
[error] 	at dotty.tools.dotc.typer.Typer.typedExpr(Typer.scala:2943)
[error] 	at dotty.tools.dotc.typer.FrontEnd.liftedTree1$1(FrontEnd.scala:79)
[error] 	at dotty.tools.dotc.typer.FrontEnd.typeCheck$$anonfun$1(FrontEnd.scala:84)
[error] 	at dotty.tools.dotc.typer.FrontEnd.monitor(FrontEnd.scala:43)
[error] 	at dotty.tools.dotc.typer.FrontEnd.typeCheck(FrontEnd.scala:85)
[error] 	at dotty.tools.dotc.typer.FrontEnd.runOn$$anonfun$3(FrontEnd.scala:120)
[error] 	at scala.runtime.function.JProcedure1.apply(JProcedure1.java:15)
[error] 	at scala.runtime.function.JProcedure1.apply(JProcedure1.java:10)
[error] 	at scala.collection.immutable.List.foreach(List.scala:333)
[error] 	at dotty.tools.dotc.typer.FrontEnd.runOn(FrontEnd.scala:120)
[error] 	at dotty.tools.dotc.Run.runPhases$4$$anonfun$4(Run.scala:205)
[error] 	at scala.runtime.function.JProcedure1.apply(JProcedure1.java:15)
[error] 	at scala.runtime.function.JProcedure1.apply(JProcedure1.java:10)
[error] 	at scala.collection.ArrayOps$.foreach$extension(ArrayOps.scala:1323)
[error] 	at dotty.tools.dotc.Run.runPhases$5(Run.scala:216)
[error] 	at dotty.tools.dotc.Run.compileUnits$$anonfun$1(Run.scala:224)
[error] 	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.scala:18)
[error] 	at dotty.tools.dotc.util.Stats$.maybeMonitored(Stats.scala:67)
[error] 	at dotty.tools.dotc.Run.compileUnits(Run.scala:231)
[error] 	at dotty.tools.dotc.Run.compileUnits(Run.scala:172)
[error] 	at dotty.tools.dotc.Driver.finish(Driver.scala:60)
[error] 	at dotty.tools.dotc.Driver.doCompile(Driver.scala:40)
[error] 	at dotty.tools.xsbt.CompilerBridgeDriver.run(CompilerBridgeDriver.java:88)
[error] 	at dotty.tools.xsbt.CompilerBridge.run(CompilerBridge.java:22)
[error] 	at sbt.internal.inc.AnalyzingCompiler.compile(AnalyzingCompiler.scala:91)
[error] 	at sbt.internal.inc.MixedAnalyzingCompiler.$anonfun$compile$7(MixedAnalyzingCompiler.scala:186)
[error] 	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
[error] 	at sbt.internal.inc.MixedAnalyzingCompiler.timed(MixedAnalyzingCompiler.scala:241)
[error] 	at sbt.internal.inc.MixedAnalyzingCompiler.$anonfun$compile$4(MixedAnalyzingCompiler.scala:176)
[error] 	at sbt.internal.inc.MixedAnalyzingCompiler.$anonfun$compile$4$adapted(MixedAnalyzingCompiler.scala:157)
[error] 	at sbt.internal.inc.JarUtils$.withPreviousJar(JarUtils.scala:239)
[error] 	at sbt.internal.inc.MixedAnalyzingCompiler.compileScala$1(MixedAnalyzingCompiler.scala:157)
[error] 	at sbt.internal.inc.MixedAnalyzingCompiler.compile(MixedAnalyzingCompiler.scala:204)
[error] 	at sbt.internal.inc.IncrementalCompilerImpl.$anonfun$compileInternal$1(IncrementalCompilerImpl.scala:528)
[error] 	at sbt.internal.inc.IncrementalCompilerImpl.$anonfun$compileInternal$1$adapted(IncrementalCompilerImpl.scala:528)
[error] 	at sbt.internal.inc.Incremental$.$anonfun$apply$5(Incremental.scala:174)
[error] 	at sbt.internal.inc.Incremental$.$anonfun$apply$5$adapted(Incremental.scala:172)
[error] 	at sbt.internal.inc.Incremental$$anon$2.run(Incremental.scala:457)
[error] 	at sbt.internal.inc.IncrementalCommon$CycleState.next(IncrementalCommon.scala:116)
[error] 	at sbt.internal.inc.IncrementalCommon$$anon$1.next(IncrementalCommon.scala:56)
[error] 	at sbt.internal.inc.IncrementalCommon$$anon$1.next(IncrementalCommon.scala:52)
[error] 	at sbt.internal.inc.IncrementalCommon.cycle(IncrementalCommon.scala:261)
[error] 	at sbt.internal.inc.Incremental$.$anonfun$incrementalCompile$8(Incremental.scala:412)
[error] 	at sbt.internal.inc.Incremental$.withClassfileManager(Incremental.scala:499)
[error] 	at sbt.internal.inc.Incremental$.incrementalCompile(Incremental.scala:399)
[error] 	at sbt.internal.inc.Incremental$.apply(Incremental.scala:166)
[error] 	at sbt.internal.inc.IncrementalCompilerImpl.compileInternal(IncrementalCompilerImpl.scala:528)
[error] 	at sbt.internal.inc.IncrementalCompilerImpl.$anonfun$compileIncrementally$1(IncrementalCompilerImpl.scala:482)
[error] 	at sbt.internal.inc.IncrementalCompilerImpl.handleCompilationError(IncrementalCompilerImpl.scala:332)
[error] 	at sbt.internal.inc.IncrementalCompilerImpl.compileIncrementally(IncrementalCompilerImpl.scala:420)
[error] 	at sbt.internal.inc.IncrementalCompilerImpl.compile(IncrementalCompilerImpl.scala:137)
[error] 	at sbt.Defaults$.compileIncrementalTaskImpl(Defaults.scala:2346)
[error] 	at sbt.Defaults$.$anonfun$compileIncrementalTask$2(Defaults.scala:2303)
[error] 	at sbt.internal.io.Retry$.apply(Retry.scala:46)
[error] 	at sbt.internal.io.Retry$.apply(Retry.scala:28)
[error] 	at sbt.internal.io.Retry$.apply(Retry.scala:23)
[error] 	at sbt.internal.server.BspCompileTask$.compute(BspCompileTask.scala:31)
[error] 	at sbt.Defaults$.$anonfun$compileIncrementalTask$1(Defaults.scala:2299)
[error] 	at scala.Function1.$anonfun$compose$1(Function1.scala:49)
[error] 	at sbt.internal.util.$tilde$greater.$anonfun$$u2219$1(TypeFunctions.scala:62)
[error] 	at sbt.std.Transform$$anon$4.work(Transform.scala:68)
[error] 	at sbt.Execute.$anonfun$submit$2(Execute.scala:282)
[error] 	at sbt.internal.util.ErrorHandling$.wideConvert(ErrorHandling.scala:23)
[error] 	at sbt.Execute.work(Execute.scala:291)
[error] 	at sbt.Execute.$anonfun$submit$1(Execute.scala:282)
[error] 	at sbt.ConcurrentRestrictions$$anon$4.$anonfun$submitValid$1(ConcurrentRestrictions.scala:265)
[error] 	at sbt.CompletionService$$anon$2.call(CompletionService.scala:64)
[error] 	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[error] 	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
[error] 	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[error] 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[error] 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[error] 	at java.base/java.lang.Thread.run(Thread.java:834)
[error] Caused by: java.lang.ClassNotFoundException: DFBitsCompanion
[error] 	at java.base/java.net.URLClassLoader.findClass(URLClassLoader.java:471)
[error] 	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:588)
[error] 	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:521)
[error] 	at java.base/java.lang.ClassLoader.defineClass1(Native Method)
[error] 	at java.base/java.lang.ClassLoader.defineClass(ClassLoader.java:1016)
[error] 	at java.base/java.security.SecureClassLoader.defineClass(SecureClassLoader.java:174)
[error] 	at java.base/java.net.URLClassLoader.defineClass(URLClassLoader.java:550)
[error] 	at java.base/java.net.URLClassLoader$1.run(URLClassLoader.java:458)
[error] 	at java.base/java.net.URLClassLoader$1.run(URLClassLoader.java:452)
[error] 	at java.base/java.security.AccessController.doPrivileged(Native Method)
[error] 	at java.base/java.net.URLClassLoader.findClass(URLClassLoader.java:451)
[error] 	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:588)
[error] 	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:521)
[error] 	at java.base/java.lang.Class.getDeclaredFields0(Native Method)
[error] 	at java.base/java.lang.Class.privateGetDeclaredFields(Class.java:3061)
[error] 	at java.base/java.lang.Class.getField0(Class.java:3246)
[error] 	at java.base/java.lang.Class.getField(Class.java:1997)
[error] 	at dotty.tools.dotc.transform.Splicer$Interpreter.loadModule(Splicer.scala:387)
[error] 	at dotty.tools.dotc.transform.Splicer$Interpreter.interpretedStaticMethodCall(Splicer.scala:356)
[error] 	at dotty.tools.dotc.transform.Splicer$Interpreter.interpretTree(Splicer.scala:260)
[error] 	at dotty.tools.dotc.transform.Splicer$Interpreter.interpretTree$$anonfun$2(Splicer.scala:281)
[error] 	at dotty.tools.dotc.transform.Splicer$.$anonfun$2(Splicer.scala:56)
[error] 	at scala.Option.fold(Option.scala:263)
[error] 	at dotty.tools.dotc.transform.Splicer$.splice(Splicer.scala:56)
[error] 	at dotty.tools.dotc.typer.Inliner.dotty$tools$dotc$typer$Inliner$$expandMacro(Inliner.scala:1633)
[error] 	at dotty.tools.dotc.typer.Inliner$InlineTyper.typedApply(Inliner.scala:1414)
[error] 	at dotty.tools.dotc.typer.Typer.typedUnnamed$1(Typer.scala:2695)
[error] 	at dotty.tools.dotc.typer.Typer.typedUnadapted(Typer.scala:2758)
[error] 	at dotty.tools.dotc.typer.ReTyper.typedUnadapted(ReTyper.scala:118)
[error] 	at dotty.tools.dotc.typer.Inliner$InlineTyper.typedUnadapted(Inliner.scala:1494)
[error] 	at dotty.tools.dotc.typer.Typer.typed(Typer.scala:2823)
[error] 	at dotty.tools.dotc.typer.Typer.typed(Typer.scala:2820)
[error] 	at dotty.tools.dotc.typer.Typer.typed(Typer.scala:2827)
[error] 	at dotty.tools.dotc.typer.Inliner.inlined(Inliner.scala:901)
[error] 	at dotty.tools.dotc.typer.Inliner$.inlineCall(Inliner.scala:153)
[error] 	at dotty.tools.dotc.typer.Typer.adaptNoArgsOther$4(Typer.scala:3504)
[error] 	at dotty.tools.dotc.typer.Typer.adaptNoArgs$1(Typer.scala:3620)
[error] 	at dotty.tools.dotc.typer.Typer.adapt1(Typer.scala:3811)
[error] 	at dotty.tools.dotc.typer.Typer.adapt(Typer.scala:3163)
[error] 	at dotty.tools.dotc.typer.Typer.readapt$1(Typer.scala:3174)
[error] 	at dotty.tools.dotc.typer.Typer.adapt1(Typer.scala:3798)
[error] 	at dotty.tools.dotc.typer.Typer.adapt(Typer.scala:3163)
[error] 	at dotty.tools.dotc.typer.Implicits.typedImplicit(Implicits.scala:1024)
[error] 	at dotty.tools.dotc.typer.Implicits.typedImplicit$(Implicits.scala:769)
[error] 	at dotty.tools.dotc.typer.Typer.typedImplicit(Typer.scala:106)
[error] 	at dotty.tools.dotc.typer.Implicits$ImplicitSearch.tryImplicit(Implicits.scala:1119)
[error] 	at dotty.tools.dotc.typer.Implicits$ImplicitSearch.rank$1(Implicits.scala:1191)
[error] 	at dotty.tools.dotc.typer.Implicits$ImplicitSearch.searchImplicit(Implicits.scala:1359)
[error] 	at dotty.tools.dotc.typer.Implicits$ImplicitSearch.searchImplicit(Implicits.scala:1366)
[error] 	at dotty.tools.dotc.typer.Implicits$ImplicitSearch.searchImplicit(Implicits.scala:1374)
[error] 	at dotty.tools.dotc.typer.Implicits$ImplicitSearch.bestImplicit(Implicits.scala:1399)
[error] 	at dotty.tools.dotc.typer.Implicits.inferImplicit(Implicits.scala:974)
[error] 	at dotty.tools.dotc.typer.Implicits.inferImplicit$(Implicits.scala:769)
[error] 	at dotty.tools.dotc.typer.Typer.inferImplicit(Typer.scala:106)
[error] 	at dotty.tools.dotc.typer.Implicits.inferImplicitArg(Implicits.scala:843)
[error] 	at dotty.tools.dotc.typer.Implicits.inferImplicitArg$(Implicits.scala:769)
[error] 	at dotty.tools.dotc.typer.Typer.inferImplicitArg(Typer.scala:106)
[error] 	at dotty.tools.dotc.typer.Typer.implicitArgs$1(Typer.scala:3285)
[error] 	at dotty.tools.dotc.typer.Typer.addImplicitArgs$3(Typer.scala:3321)
[error] 	at dotty.tools.dotc.typer.Typer.adaptNoArgsImplicitMethod$2(Typer.scala:3401)
[error] 	at dotty.tools.dotc.typer.Typer.adaptNoArgs$1(Typer.scala:3598)
[error] 	at dotty.tools.dotc.typer.Typer.adapt1(Typer.scala:3811)
[error] 	at dotty.tools.dotc.typer.Typer.adapt(Typer.scala:3163)
[error] 	at dotty.tools.dotc.typer.Typer.typed(Typer.scala:2823)
[error] 	at dotty.tools.dotc.typer.Applications.extMethodApply(Applications.scala:2225)
[error] 	at dotty.tools.dotc.typer.Applications.extMethodApply$(Applications.scala:316)
[error] 	at dotty.tools.dotc.typer.Typer.extMethodApply(Typer.scala:106)
[error] 	at dotty.tools.dotc.typer.Typer.tryExtension$1(Typer.scala:3077)
[error] 	at dotty.tools.dotc.typer.Typer.tryExtensionOrConversion(Typer.scala:3088)
[error] 	at dotty.tools.dotc.typer.Typer.typedSelect(Typer.scala:583)
[error] 	at dotty.tools.dotc.typer.Typer.typeSelectOnTerm$1(Typer.scala:622)
[error] 	at dotty.tools.dotc.typer.Typer.typedSelect(Typer.scala:659)
[error] 	at dotty.tools.dotc.typer.Typer.typedNamed$1(Typer.scala:2665)
[error] 	at dotty.tools.dotc.typer.Typer.typedUnadapted(Typer.scala:2757)
[error] 	at dotty.tools.dotc.typer.Typer.typed(Typer.scala:2823)
[error] 	at dotty.tools.dotc.typer.Typer.typed(Typer.scala:2827)
[error] 	at dotty.tools.dotc.typer.Typer.typedExpr(Typer.scala:2943)
[error] 	at dotty.tools.dotc.typer.Typer.$anonfun$34(Typer.scala:2168)
[error] 	at dotty.tools.dotc.typer.PrepareInlineable$.dropInlineIfError(PrepareInlineable.scala:225)
[error] 	at dotty.tools.dotc.typer.Typer.typedDefDef(Typer.scala:2168)
[error] 	at dotty.tools.dotc.typer.Typer.typedNamed$1(Typer.scala:2672)
[error] 	at dotty.tools.dotc.typer.Typer.typedUnadapted(Typer.scala:2757)
[error] 	at dotty.tools.dotc.typer.Typer.typed(Typer.scala:2823)
[error] 	at dotty.tools.dotc.typer.Typer.typed(Typer.scala:2827)
[error] 	at dotty.tools.dotc.typer.Typer.traverse$1(Typer.scala:2849)
[error] 	at dotty.tools.dotc.typer.Typer.typedStats(Typer.scala:2899)
[error] 	at dotty.tools.dotc.typer.Typer.typedClassDef(Typer.scala:2354)
[error] 	at dotty.tools.dotc.typer.Typer.typedTypeOrClassDef$2(Typer.scala:2683)
[error] 	at dotty.tools.dotc.typer.Typer.typedNamed$1(Typer.scala:2687)
[error] 	at dotty.tools.dotc.typer.Typer.typedUnadapted(Typer.scala:2757)
[error] 	at dotty.tools.dotc.typer.Typer.typed(Typer.scala:2823)
[error] 	at dotty.tools.dotc.typer.Typer.typed(Typer.scala:2827)
[error] 	at dotty.tools.dotc.typer.Typer.traverse$1(Typer.scala:2849)
[error] 	at dotty.tools.dotc.typer.Typer.typedStats(Typer.scala:2899)
[error] 	at dotty.tools.dotc.typer.Typer.typedPackageDef(Typer.scala:2477)
[error] 	at dotty.tools.dotc.typer.Typer.typedUnnamed$1(Typer.scala:2728)
[error] 	at dotty.tools.dotc.typer.Typer.typedUnadapted(Typer.scala:2758)
[error] 	at dotty.tools.dotc.typer.Typer.typed(Typer.scala:2823)
[error] 	at dotty.tools.dotc.typer.Typer.typed(Typer.scala:2827)
[error] 	at dotty.tools.dotc.typer.Typer.traverse$1(Typer.scala:2876)
[error] 	at dotty.tools.dotc.typer.Typer.typedStats(Typer.scala:2899)
[error] 	at dotty.tools.dotc.typer.Typer.typedPackageDef(Typer.scala:2477)
[error] 	at dotty.tools.dotc.typer.Typer.typedUnnamed$1(Typer.scala:2728)
[error] 	at dotty.tools.dotc.typer.Typer.typedUnadapted(Typer.scala:2758)
[error] 	at dotty.tools.dotc.typer.Typer.typed(Typer.scala:2823)
[error] 	at dotty.tools.dotc.typer.Typer.typed(Typer.scala:2827)
[error] 	at dotty.tools.dotc.typer.Typer.typedExpr(Typer.scala:2943)
[error] 	at dotty.tools.dotc.typer.FrontEnd.liftedTree1$1(FrontEnd.scala:79)
[error] 	at dotty.tools.dotc.typer.FrontEnd.typeCheck$$anonfun$1(FrontEnd.scala:84)
[error] 	at dotty.tools.dotc.typer.FrontEnd.monitor(FrontEnd.scala:43)
[error] 	at dotty.tools.dotc.typer.FrontEnd.typeCheck(FrontEnd.scala:85)
[error] 	at dotty.tools.dotc.typer.FrontEnd.runOn$$anonfun$3(FrontEnd.scala:120)
[error] 	at scala.runtime.function.JProcedure1.apply(JProcedure1.java:15)
[error] 	at scala.runtime.function.JProcedure1.apply(JProcedure1.java:10)
[error] 	at scala.collection.immutable.List.foreach(List.scala:333)
[error] 	at dotty.tools.dotc.typer.FrontEnd.runOn(FrontEnd.scala:120)
[error] 	at dotty.tools.dotc.Run.runPhases$4$$anonfun$4(Run.scala:205)
[error] 	at scala.runtime.function.JProcedure1.apply(JProcedure1.java:15)
[error] 	at scala.runtime.function.JProcedure1.apply(JProcedure1.java:10)
[error] 	at scala.collection.ArrayOps$.foreach$extension(ArrayOps.scala:1323)
[error] 	at dotty.tools.dotc.Run.runPhases$5(Run.scala:216)
[error] 	at dotty.tools.dotc.Run.compileUnits$$anonfun$1(Run.scala:224)
[error] 	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.scala:18)
[error] 	at dotty.tools.dotc.util.Stats$.maybeMonitored(Stats.scala:67)
[error] 	at dotty.tools.dotc.Run.compileUnits(Run.scala:231)
[error] 	at dotty.tools.dotc.Run.compileUnits(Run.scala:172)
[error] 	at dotty.tools.dotc.Driver.finish(Driver.scala:60)
[error] 	at dotty.tools.dotc.Driver.doCompile(Driver.scala:40)
[error] 	at dotty.tools.xsbt.CompilerBridgeDriver.run(CompilerBridgeDriver.java:88)
[error] 	at dotty.tools.xsbt.CompilerBridge.run(CompilerBridge.java:22)
[error] 	at sbt.internal.inc.AnalyzingCompiler.compile(AnalyzingCompiler.scala:91)
[error] 	at sbt.internal.inc.MixedAnalyzingCompiler.$anonfun$compile$7(MixedAnalyzingCompiler.scala:186)
[error] 	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
[error] 	at sbt.internal.inc.MixedAnalyzingCompiler.timed(MixedAnalyzingCompiler.scala:241)
[error] 	at sbt.internal.inc.MixedAnalyzingCompiler.$anonfun$compile$4(MixedAnalyzingCompiler.scala:176)
[error] 	at sbt.internal.inc.MixedAnalyzingCompiler.$anonfun$compile$4$adapted(MixedAnalyzingCompiler.scala:157)
[error] 	at sbt.internal.inc.JarUtils$.withPreviousJar(JarUtils.scala:239)
[error] 	at sbt.internal.inc.MixedAnalyzingCompiler.compileScala$1(MixedAnalyzingCompiler.scala:157)
[error] 	at sbt.internal.inc.MixedAnalyzingCompiler.compile(MixedAnalyzingCompiler.scala:204)
[error] 	at sbt.internal.inc.IncrementalCompilerImpl.$anonfun$compileInternal$1(IncrementalCompilerImpl.scala:528)
[error] 	at sbt.internal.inc.IncrementalCompilerImpl.$anonfun$compileInternal$1$adapted(IncrementalCompilerImpl.scala:528)
[error] 	at sbt.internal.inc.Incremental$.$anonfun$apply$5(Incremental.scala:174)
[error] 	at sbt.internal.inc.Incremental$.$anonfun$apply$5$adapted(Incremental.scala:172)
[error] 	at sbt.internal.inc.Incremental$$anon$2.run(Incremental.scala:457)
[error] 	at sbt.internal.inc.IncrementalCommon$CycleState.next(IncrementalCommon.scala:116)
[error] 	at sbt.internal.inc.IncrementalCommon$$anon$1.next(IncrementalCommon.scala:56)
[error] 	at sbt.internal.inc.IncrementalCommon$$anon$1.next(IncrementalCommon.scala:52)
[error] 	at sbt.internal.inc.IncrementalCommon.cycle(IncrementalCommon.scala:261)
[error] 	at sbt.internal.inc.Incremental$.$anonfun$incrementalCompile$8(Incremental.scala:412)
[error] 	at sbt.internal.inc.Incremental$.withClassfileManager(Incremental.scala:499)
[error] 	at sbt.internal.inc.Incremental$.incrementalCompile(Incremental.scala:399)
[error] 	at sbt.internal.inc.Incremental$.apply(Incremental.scala:166)
[error] 	at sbt.internal.inc.IncrementalCompilerImpl.compileInternal(IncrementalCompilerImpl.scala:528)
[error] 	at sbt.internal.inc.IncrementalCompilerImpl.$anonfun$compileIncrementally$1(IncrementalCompilerImpl.scala:482)
[error] 	at sbt.internal.inc.IncrementalCompilerImpl.handleCompilationError(IncrementalCompilerImpl.scala:332)
[error] 	at sbt.internal.inc.IncrementalCompilerImpl.compileIncrementally(IncrementalCompilerImpl.scala:420)
[error] 	at sbt.internal.inc.IncrementalCompilerImpl.compile(IncrementalCompilerImpl.scala:137)
[error] 	at sbt.Defaults$.compileIncrementalTaskImpl(Defaults.scala:2346)
[error] 	at sbt.Defaults$.$anonfun$compileIncrementalTask$2(Defaults.scala:2303)
[error] 	at sbt.internal.io.Retry$.apply(Retry.scala:46)
[error] 	at sbt.internal.io.Retry$.apply(Retry.scala:28)
[error] 	at sbt.internal.io.Retry$.apply(Retry.scala:23)
[error] 	at sbt.internal.server.BspCompileTask$.compute(BspCompileTask.scala:31)
[error] 	at sbt.Defaults$.$anonfun$compileIncrementalTask$1(Defaults.scala:2299)
[error] 	at scala.Function1.$anonfun$compose$1(Function1.scala:49)
[error] 	at sbt.internal.util.$tilde$greater.$anonfun$$u2219$1(TypeFunctions.scala:62)
[error] 	at sbt.std.Transform$$anon$4.work(Transform.scala:68)
[error] 	at sbt.Execute.$anonfun$submit$2(Execute.scala:282)
[error] 	at sbt.internal.util.ErrorHandling$.wideConvert(ErrorHandling.scala:23)
[error] 	at sbt.Execute.work(Execute.scala:291)
[error] 	at sbt.Execute.$anonfun$submit$1(Execute.scala:282)
[error] 	at sbt.ConcurrentRestrictions$$anon$4.$anonfun$submitValid$1(ConcurrentRestrictions.scala:265)
[error] 	at sbt.CompletionService$$anon$2.call(CompletionService.scala:64)
[error] 	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[error] 	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
[error] 	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[error] 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[error] 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[error] 	at java.base/java.lang.Thread.run(Thread.java:834)
[error] (core / Compile / compileIncremental) java.lang.NoClassDefFoundError: DFBitsCompanion
```
</details>

--- a/compiler/src/dotty/tools/dotc/transform/Splicer.scala
+++ b/compiler/src/dotty/tools/dotc/transform/Splicer.scala
@@ -350,12 +350,18 @@ object Splicer {
 
     private def interpretedStaticMethodCall(moduleClass: Symbol, fn: Symbol)(implicit env: Env): List[Object] => Object = {
       val (inst, clazz) =
-        if (moduleClass.name.startsWith(str.REPL_SESSION_LINE))
-          (null, loadReplLineClass(moduleClass))
-        else {
-          val inst = loadModule(moduleClass)
-          (inst, inst.getClass)
-        }
+        try
+          if (moduleClass.name.startsWith(str.REPL_SESSION_LINE))
+            (null, loadReplLineClass(moduleClass))
+          else {
+            val inst = loadModule(moduleClass)
+            (inst, inst.getClass)
+          }
+        catch
+          case MissingClassDefinedInCurrentRun(sym)  if ctx.compilationUnit.isSuspendable =>
+            if (ctx.settings.XprintSuspension.value)
+              report.echo(i"suspension triggered by a dependency on $sym", pos)
+            ctx.compilationUnit.suspend() // this throws a SuspendException
 
       val name = fn.name.asTermName
       val method = getMethod(clazz, name, paramsSig(fn))

--- a/tests/neg-macros/i12788/Macro.scala
+++ b/tests/neg-macros/i12788/Macro.scala
@@ -1,0 +1,21 @@
+import scala.quoted.*
+
+class InlinedInt[T <: Int]
+
+object DFType:
+  trait Width[T]:
+    type Out <: Int
+  transparent inline given [T]: Width[T] = ${ getWidthMacro[T] }
+  def getWidthMacro[T](using Quotes, Type[T]): Expr[Width[T]] =
+    '{
+      new Width[T] {
+        type Out = 1
+      }
+    }
+
+  trait DFBits[W <: Int]
+  object DFBits extends DFBitsCompanion
+
+trait DFToken[T]
+extension [T](token: DFToken[T])(using w: DFType.Width[T])
+  def width: InlinedInt[w.Out] = ???

--- a/tests/neg-macros/i12788/Test.scala
+++ b/tests/neg-macros/i12788/Test.scala
@@ -1,0 +1,4 @@
+// nopos-error
+trait DFBitsCompanion:
+  type Token[W <: Int] = DFToken[DFType.DFBits[W]]
+  extension [LW <: Int](lhs: DFType.DFBits.Token[LW]) def foo: Unit = lhs.width

